src/app/components/Icons/BackButtonIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+function BackButtonIcon(): JSX.Element {
+  return (
+    <svg width="12.828" height="22.828" viewBox="0 0 12.828 22.828">
+      <g
+        id="Gruppe_346"
+        data-name="Gruppe 346"
```

Comment:
Usually, you can remove all `id` and `data-name` from your SVGs

Reply:
thanks!